src/components/SecondaryNav/SecondaryNav.css

```diff
@@ -0,0 +1,50 @@
+.SecondaryNav {
+  @apply border-b-2
+      relative
+      bg-white
+      overflow-hidden
+      border-mediumGrey;
+}
+
+.SecondaryNav ul {
+  @apply p-0
+      m-0
+      relative
+      inline-block
+      h-full
+      max-w-full;
+}
+
+.SecondaryNav--link {
+  @apply m-0
+      p-0
+      relative
+      inline-block
+      h-full
+      border-lightGrey
+      border
+      border-b-0
+      font-semibold
+      cursor-pointer;
+
+  a {
```

Comment:
move these styles up to the `SecondaryNav--link` , since we are using button elements we don't need to use anchor tags since we will navigate using the `onClick` prop on the button